### PR TITLE
Remove Algolia credentials from .travis.yml file

### PR DESCRIPTION
**Pull request by julienbourdeau**
Credentials are now defined in the Travis admin.

### CODE REVIEW

File: Tests/Indexer/Indexer.php [hunk 1]
```diff
@@ -66,6 +66,7 @@ public function setApiSettings($application_id, $api_key, $connection_timeout =
             if ($kernel->getContainer()->getParameter('algolia.get_credentials_from_env')) {
                 $application_id = getenv('ALGOLIA_APPLICATION_ID');
                 $api_key = getenv('ALGOLIA_API_KEY');
+                var_dump($application_id, $api_key); die;
```
Comment (JanPetr) [inline Tests/Indexer/Indexer.php]: It'll leak the API credentials to publicly accessible Travis build log and you'll need to regenerate API key

Comment (julienbourdeau) [inline Tests/Indexer/Indexer.php]: Travis automatically hide them, but yeah that's not a good way to go ^^ I should have used `strlen` for instance.

**Comment by alcaeus** [comment]
@julienbourdeau does this change anything about pull requests from forks being built?

**Comment by julienbourdeau** [comment]
Unfortunately, I don't think so :(

**Pull request merged**